src/test/java/org/sonatype/cs/metrics/SuccessMetricsWebApplicationTest.java

```diff
@@ -0,0 +1,111 @@
+package org.sonatype.cs.metrics;
```

Comment:
There is probably a better way to do these integration tests instead of saving all of those really long files, but I would be open to just creating a new tech debt story to figure out what that is.

Reply:
Great, let's do that